packages/lodestar/src/network/discovery/discv5.ts

```diff
@@ -0,0 +1,75 @@
+import {EventEmitter} from "events";
+import randombytes from "randombytes";
+import PeerInfo from "peer-info";
+import Multiaddr from "multiaddr";
+import {
+  Discv5,
+  ENR,
+  createNodeId,
+} from "@chainsafe/discv5";
+
+export interface IDiscv5DiscoveryInputOptions {
+  enr: ENR;
+  bindAddr: Multiaddr;
+  bootEnrs: ENR[];
+}
+
+export interface IDiscv5DiscoveryOptions extends IDiscv5DiscoveryInputOptions {
+  peerInfo: PeerInfo;
+}
+
+/**
+ * Discv5Discovery
+ */
+export class Discv5Discovery extends EventEmitter {
+  static tag = "discv5";
+
+  private discv5: Discv5;
+  private started: NodeJS.Timer | boolean;
+
+  constructor(options: IDiscv5DiscoveryOptions) {
+    super();
+    this.discv5 = Discv5.create(options.enr, options.peerInfo.id, options.bindAddr);
+    options.bootEnrs.forEach((bootEnr) => this.discv5.addEnr(bootEnr));
+  }
+
+  async start(): Promise<void> {
+    if (this.started) {
+      return;
+    }
+    this.started = true;
+    await this.discv5.start();
+    setTimeout(() => this.findPeers, 1);
```

Comment:
Why wait 1ms before finding peers?

Reply:
We don't need to, just was using the timeout to run asyncronously. 
Will replace this line with `this.findPeers();`